Software/grab/GrabberBase.cpp

```diff
@@ -167,6 +167,30 @@ void GrabberBase::grab()
 			// Convert coordinates from "Main" desktop coord-system to capture-monitor coord-system
 			QRect preparedRect = clippedRect.translated(-monitorRect.x(), -monitorRect.y());
 
+			// grabbed screen is rotated => rotate the widget
+			if (grabbedScreen->rotation != 0)
+				if (grabbedScreen->rotation % 4 == 1) // rotated 90
```

Comment:
isn't `3` the max for `rotation` and thus `% 4` isn't doing much?

Reply:
Well yes. This is defensive programming - the rotation is uchar, defined as mulitples of 90 degress. In case someone (else who later develops another grabber) sets the rotation to 5 for whatever reason, it still works.
The alternative would be to use an enum, which ensures only those 4 values can be used.

Reply:
I'm not sure if it rotates the same way, but maybe [this](https://doc.qt.io/qt-5/qscreen.html#mapBetween) can be leveraged along with qt orientations?

Reply:
Probably. It requires keeping track of both orientations though. Feel free to send a PR if you feel that's cleaner...
BTW, I'm sorry I didn't make a release with your new moodlights, will publish one today.

Reply:
Oh yeah! Since I was running master builds I forgot about it until few days ago. Well that'll make it slightly less under-the-hood-only-changes :)